docs/schema.md

```diff
@@ -155,20 +155,156 @@ string of the format `<DB>-<ENTRYID>`, where `DB` names the database and
 
 The defined database prefixes and their "home" databases are:
 
-| Prefix | Description |
-| ------ | ----------- |
-| `A` | The [Android vulnerability database](TODO). Serving the shared format [here](https://storage.googleapis.com/android-osv/). |
-| `GO` | The [Go vulnerability database](https://github.com/golang/vulndb). Serving the shared format [here](https://storage.googleapis.com/go-vulndb/). |
-| `OSV` | The <https://osv.dev> vulnerability database. Serving `<ID>` in the shared format at `https://api.osv.dev/v1/vulns/<ID>` |
-| `PYSEC` | The [PyPI vulnerability database](https://github.com/pypa/advisory-db). Serving `<ID>` in the shared format at `https://api.osv.dev/v1/vulns/<ID>` |
-| `RUSTSEC` | [The Rust crates vulnerability database](https://github.com/rustsec/advisory-db). Serving `<ID>` in the shared format at  `https://github.com/RustSec/advisory-db/blob/osv/crates/<ID>.json` |
-| `GSD` | The GSD database. Serving the shared format [here](https://github.com/cloudsecurityalliance/gsd-database). |
-| `GHSA` | The GitHub Security Advisory database. Serving the shared format [here](https://github.com/github/advisory-database). |
-| `LBSEC` | The LoopBack Advisory Database. Serving the shared format [here](https://github.com/loopbackio/security/tree/main/advisories). |
-| `DSA`/`DLA`/`DTSA` | The [Debian Security Advisory](https://www.debian.org/security/) database. Serving the shared format [here](https://storage.googleapis.com/debian-osv/dsa-osv/). |
-| `RLSA`/`RXSA` | The [Rocky Linux Security Advisory](https://errata.rockylinux.org) database. Serving the shared format [here](https://apollo.build.resf.org/api/v3/osv/)
-| `ALSA`/`ALBA`/`ALEA` | The [AlmaLinux Security Advisory](https://errata.almalinux.org/). Serving the shared format [here](https://github.com/AlmaLinux/osv-database)
-| Your database here | [Send us a PR](https://github.com/ossf/osv-schema/compare). |
+<!-- Unfortunately, markdown tables are even worse than this to read/write -->
```

Comment:
Here is what this looks like when rendered out:

<!-- Unfortunately, markdown tables are even worse than this to read/write -->
<table>
  <thead>
    <tr>
      <th>Prefix</th>
      <th>Database</th>
      <th>Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><code>A</code></td>
      <td><a href="https://storage.googleapis.com/android-osv/">Android Vulnerability Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>N/A</code></li>
          <li>ID API Endpoint: <code>https://storage.googleapis.com/android-osv/&lt;ID&gt;.json</code></li>
          <li>ID Regex: <code>/^(ASB-A|A)-\d{4,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>GO</code></td>
      <td><a href="https://pkg.go.dev/vuln/">Go Vulnerability Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://pkg.go.dev/vuln/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>https://vuln.go.dev/ID/&lt;ID&gt;.json</code></li>
          <li>ID Regex: <code>/^GO-\d{4}-\d{4,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>OSV</code></td>
      <td><a href="https://osv.dev/list">OSV Vulnerability Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://osv.dev/vulnerability/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>https://api.osv.dev/v1/vulns/&lt;ID&gt;</code></li>
          <li>ID Regex: <code>/^OSV-\d{4}-\d{1,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>PYSEC</code></td>
      <td><a href="https://github.com/pypa/advisory-db">PyPI Vulnerability Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://osv.dev/vulnerability/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>https://api.osv.dev/v1/vulns/&lt;ID&gt;</code></li>
          <li>ID Regex: <code>/^PYSEC-\d{4}-\d{1,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>RUSTSEC</code></td>
      <td><a href="https://github.com/rustsec/advisory-db">RustSec Advisory Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://rustsec.org/advisories/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>https://raw.githubusercontent.com/rustsec/advisory-db/osv/crates/&lt;ID&gt;.json</code></li>
          <li>ID Regex: <code>/^RUSTSEC-\d{4}-\d{4,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>GSD</code></td>
      <td><a href="https://github.com/cloudsecurityalliance/gsd-database">Global Security Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://gsd.id/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>https://api.gsd.id/&lt;ID&gt;</code></li>
          <li>ID Regex: <code>/^GSD-\d{4}-\d{4,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>GHSA</code></td>
      <td><a href="https://github.com/github/advisory-database">GitHub Security Advisory Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://github.com/advisories/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>N/A</code></li>
          <li>ID Regex: <code>/^GHSA-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>LBSEC</code></td>
      <td><a href="https://github.com/loopbackio/security/tree/main/advisories">LoopBack Advisory Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>N/A</code></li>
          <li>ID API Endpoint: <code>N/A</code></li>
          <li>ID Regex: <code>/^lbsa-\d{8}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>DSA</code>/<code>DLA</code>/<code>DTSA</code></td>
      <td><a href="https://www.debian.org/security/">Debian Security Advisory Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://debian.org/security/&lt;YEAR&gt;/&lt;ID&gt;</code> (lowercase dsa)</li>
          <li>ID API Endpoint: <code>N/A</code></li>
          <li>ID Regex: <code>/^DSA-\d{4}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>RLSA</code>/<code>RXSA</code></td>
      <td><a href="https://errata.rockylinux.org">Rocky Linux Security Advisory Database</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://errata.rockylinux.org/&lt;ID&gt;</code></li>
          <li>ID API Endpoint: <code>https://apollo.build.resf.org/api/v3/osv/&lt;ID&gt;</code></li>
          <li>ID Regex: <code>/^RLSA-\d{4}:\d{4,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><code>ALSA</code>/<code>ALBA</code>/<code>ALEA</code></td>
      <td><a href="https://errata.almalinux.org/">AlmaLinux Security Advisory</a></td>
      <td>
        <ul>
          <li>How to contribute: TBD</li>
          <li>ID Canonical URL: <code>https://errata.almalinux.org/&lt;alma version&gt;/&lt;ID&gt;.html</code></li>
          <li>ID API Endpoint: <code>N/A</code></li>
          <li>ID Regex: <code>/^ALSA-\d{4}:\d{4,}$/</code></li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>Your database here</td>
      <td colspan="2"><a href="https://github.com/ossf/osv-schema/compare">Send us a PR</a></td>
    </tr>
  </tbody>
</table>

Reply:
@oliverchang How does this look?

Reply:
I can replace ID Regex with an Example ID or remove it if needed. I don't want that to be an unnecessary sticking point.

Reply:
Thanks, this looks pretty great! I'd still remove the Regex/Example ID since I don't think it provides a lot of value over just stating the prefix. 

I'd also replace "ID Canonical URL" with "Source URL" and "ID API Endpoint" with "OSV formatted URL" to be clearer.

Reply:
@oliverchang Updated!